.cirrus.yml

```diff
@@ -1,5 +1,12 @@
-container:
+gcp_credentials: ENCRYPTED[987a78af29b91ce8489594c9ab3fec21845bbe5ba68294b8f6def3cf0d380830f06687a89ea69c87344c5ade369700fe]
+
+gke_container:
   image: gcr.io/flutter-cirrus/build-engine-image:0.1
+  cluster_name: build-16-cluster
+  zone: us-central1-a
+  namespace: default
+  cpu: 14 # can't use all 16-cores; system pods needs cores too
+  memory: 60Gb
```

Comment:
Forgot to mention that the same applies for memory. you can't use all of `n1-standard-16`'s memory.

Reply:
One more question @fkorotkov : I recently updated the container image and expect Cirrus to pickup the new image by checking the digest. But it seems that Cirrus won't update the image unless that I change the image name or tag?

Reply:
It seems that I have to create a 'latest' tag in order to use it. Is this a special tag that Cirrus will always try to pull it if the digest is different, instead of using the image with the same name?

Reply:
@fkorotkov it seems that Cirrus won't pull the latest 'latest' tag image if I updated it.

Reply:
This tag affect which pull policy will be picked for a Kubernetes Job that represents the task. 

By deafuult it's IfNotPresented which means Kunernetes won't pull an image of a particular tag is already pulled. latest tag will set the policy to Always which should always check for an updated image.